src/utils/attribute.ts

```
@@ -0,0 +1,44 @@
+  if (value.includes('em')) return 'em';
+  if (value.includes('vh')) return 'vh';
+  if (value.includes('vw')) return 'vw';
+  if (value.includes('px')) return 'px';
+  return null;
+};
+
+export const stringValueWithUnit = (value?: string | number) => {
+  if (!value) return '';
+  const stringValue = typeof value === 'number' ? `${value}px` : value;
+  const unit = checkStringUnit(stringValue);
+  return `${stringValue}${unit === null ? (isNaN(Number(stringValue)) ? unit : 'px') : ''}`;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)